lib/constants.js

```diff
@@ -43,8 +43,12 @@ const whiteSpace = system({
 });
 const TYPOGRAPHY = compose(styledSystem__namespace.typography, whiteSpace);
 // Border props
-compose(styledSystem__namespace.border, styledSystem__namespace.shadow);
+const BORDER = compose(styledSystem__namespace.border, styledSystem__namespace.shadow);
```

Comment:
compose used to called without assigning to anything, that's strange?!

also, were we not exporting border and layout?

Reply:
I think what happens when `BORDER` is removed is that it considers `compose` and if it is pure. Looking at `styled-system`, it doesn't have `sideEffects` listed so it can't assume `compose` is pure and has to leave it in case of side-effects.